app/allocation/controllers/cancel.js

```diff
@@ -0,0 +1,21 @@
+const FormWizardController = require('../../../common/controllers/form-wizard')
```

Comment:
Can we add a unit test for this file please.

Reply:
Done 👍